python/paddle/optimizer/lr.py

```diff
@@ -2175,7 +2177,9 @@ class CyclicLR(LRScheduler):
 
             >>> # train on default dynamic graph mode
             >>> linear = paddle.nn.Linear(10, 10)
-            >>> scheduler = paddle.optimizer.lr.CyclicLR(base_learning_rate=0.5, max_learning_rate=1.0, step_size_up=15, step_size_down=5, verbose=True)
+            >>> scheduler = paddle.optimizer.lr.CyclicLR(
+            ...     base_learning_rate=0.5, max_learning_rate=1.0, step_size_up=15, step_size_down=5, verbose=True
```

Comment:
```suggestion
            ...     base_learning_rate=0.5, max_learning_rate=1.0, step_size_up=15, step_size_down=5, verbose=True,
```

那这个也重新格式化一下吧，5 个参数有点多，一行看不清

Reply:
好的已修改